testing/mongo-c-driver/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
+pkgname=mongo-c-driver
+pkgver=1.14.0
+pkgrel=0
+pkgdesc="Client library written in C for MongoDB"
+url="https://github.com/mongodb/mongo-c-driver"
+arch="all"
+license="Apache-2.0"
+makedepends="openssl-dev cyrus-sasl-dev icu-dev py3-sphinx cmake"
+subpackages="${pkgname}-libs-static ${pkgname}-libs ${pkgname}-dev ${pkgname}-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/mongodb/${pkgname}/archive/${pkgver}.tar.gz"
+
+build() {
+	cmake \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX:PATH=/usr \
+		-DCMAKE_INSTALL_LIBDIR:PATH=lib \
+		-DENABLE_BSON:STRING=ON \
+		-DENABLE_MONGOC:BOOL=ON \
+		-DENABLE_SSL:STRING=OPENSSL \
+		-DENABLE_AUTOMATIC_INIT_AND_CLEANUP:BOOL=OFF \
+		-DENABLE_MAN_PAGES:BOOL=ON \
+		-DENABLE_TESTS:BOOL=ON \
+		-DENABLE_EXAMPLES:BOOL=OFF \
+		-DSPHINX_EXECUTABLE:STRING=/usr/bin/sphinx-build-3 \
+		-DCMAKE_SKIP_RPATH:BOOL=ON \
+		-DENABLE_SHM_COUNTERS:BOOL=ON \
+		-DENABLE_SASL:STRING=CYRUS \
+		-DENABLE_ICU:STRING=ON \
+		-DENABLE_CRYPTO_SYSTEM_PROFILE:BOOL=ON \
+		-DENABLE_UNINSTALL:BOOL=OFF \
+		-DBUILD_VERSION:STRING=${pkgver} \
+		.
+	make
+}
+
+check() {
+	export MONGOC_TEST_SKIP_MOCK=on
+	export MONGOC_TEST_SKIP_SLOW=on
+	export MONGOC_TEST_SKIP_LIVE=on
+	make check
+}
+
+package() {
+	make DESTDIR="${pkgdir}" install
+	mkdir -p "${pkgdir}/usr/share/doc"
+	mv "${pkgdir}/usr/share/${pkgname}" "${pkgdir}/usr/share/doc"
+}
+
+libbson() {
```

Comment:
This function can be removed now that there is no `libbson` subpackage.

Reply:
Fixed.
I will mark this conversation as resolved.